pulsar-client-admin-api/src/main/java/org/apache/pulsar/client/admin/Transactions.java

```diff
@@ -121,35 +121,41 @@ TransactionInPendingAckStats getTransactionInPendingAckStats(TxnID txnID, String
      * Get transaction buffer stats.
      *
      * @param topic the topic of getting transaction buffer stats
+     * @param  lowWaterMarks Whether to get information about lowWaterMarks stored in transaction pending ack.
      * @return the future stats of transaction buffer in topic.
      */
-    CompletableFuture<TransactionBufferStats> getTransactionBufferStatsAsync(String topic);
+    CompletableFuture<TransactionBufferStats> getTransactionBufferStatsAsync(String topic, boolean lowWaterMarks);
```

Comment:
please keep the old method, you can add a "default" implementation that calls the new method, otherwise this is a breaking API change